src/fsharp/FSharp.Core/list.fsi

```diff
@@ -338,6 +632,28 @@ namespace Microsoft.FSharp.Collections
         /// <param name="state">The initial state.</param>
         ///
         /// <returns>The state object after the folding function is applied to each element of the list.</returns>
+        /// 
+        /// <example id="foldback-example-1"> Shoping for fruits hungry, you tend to take more of each as the hunger grows
```

Comment:
Typo - Shopping.

Reply:
I think an example where the state/accumulator is not a list would be better.

Reply:
I added another example before this one